vhost-device-vsock/src/vsock_conn.rs

```diff
@@ -345,6 +345,16 @@ impl<S: AsRawFd + ReadVolatile + Write + WriteVolatile + IsHybridVsock> VsockCon
         }
 
         if written_count != buf.len() {
+            // Try to re-enable EPOLLOUT in case it is disabled when txbuf is empty.
+            if VhostUserVsockThread::epoll_modify(
+                self.epoll_fd,
+                self.stream.as_raw_fd(),
+                epoll::Events::EPOLLIN | epoll::Events::EPOLLOUT,
+            )
+            .is_err()
+            {
+                error!("epoll_modify failed");
```

Comment:
....and an error here, can we do the same thing on both? 

And also put a better message, like in this case "Unable to re-enable EPOLLOUT" or something like that.